search-cache-pipeline.yml

```diff
@@ -45,22 +45,24 @@ steps:
     version: 5.0.300
     installationPath: $(Build.SourcesDirectory)/.dotnet
 
-- bash: |
-    SdkVersion=$(dotnet msbuild $(Build.SourcesDirectory)/eng/Versions.props -verbosity:diagnostic | grep "VSRedistCommonNetCoreToolsetx64PackageVersion = " | xargs | cut -d' ' -f 3)
-    echo "##vso[task.setvariable variable=RepoSdkVersion;]$SdkVersion"
-  displayName: 'Determine SDK version'
-  failOnStderr: true
+- task: UseDotNet@2
+  displayName: 'Use .NET 6.0.100'
+  inputs:
+    packageType: sdk
+    version: 6.0.100
+    installa6ionPath: $(Build.SourcesDirectory)/.dotnet
 
 - bash: >
     curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin
-    -Version $(RepoSdkVersion)
+    -Channel 7.0.1xx
```

Comment:
Do we set the `RepoSdkVersion` based on `VSRedistCommonNetCoreToolsetx64PackageVersion` and that's the reason why we cannot use it now on?
Can we use any other referenced package to set the sdk version dynamically (e.g. `MicrosoftNETCoreAppRuntimewinx64PackageVersion`)? Current approach requires updates with each preview major version, however it's not enforced so can be easily forgotten.

Reply:
I would like to avoid extra dependency for that. Each circular dependency is heavy load + additional PRs.
Imo acquiring latest daily build is good enough and allows to avoid dependency.